Packs/CommonScripts/Scripts/GetIndicatorDBotScore/GetIndicatorDBotScore.py

```diff
@@ -2,7 +2,8 @@
 from CommonServerPython import *
 from CommonServerUserPython import *
 
-INDICATOR_TYPE_DICTIONARY = {
+DbotScoreKey = 'DBotScore(val.Indicator && val.Indicator == obj.Indicator && val.Vendor == obj.Vendor && val.Type == obj.Type)'
+indicator_types = {
```

Comment:
use `DBotScoreType` for the values
```suggestion
INDICATOR_TYPES = {
```